src/openrct2/core/Guard.hpp

```diff
@@ -23,6 +23,7 @@ extern "C" {
 #endif // __cplusplus
 
 void openrct2_assert_va(bool expression, const char * message, va_list va);
+inline void openrct2_assert(bool expression, const char * message, ...);
```

Comment:
`static inline` would of course also work, then also the explicit declaration is not required. Would you prefer this over the "normal" `inline` (which definitely needs the declaration)? If so, I can certainly adapt the PR accordingly.